docs/reference/manifest/appdomain.md

```diff
@@ -18,13 +21,17 @@ Specifies additional domains that load pages in the add-in window. It also lists
 ```
 
 > [!IMPORTANT]
-> 1. The value of the **AppDomain** element must include the protocol (e.g., `<AppDomain>https://myappdomain</AppDomain>`).
-> 2. Do *not* put a closing slash, "/", on the value.
+> 1. The value of the **AppDomain** element must include the protocol (e.g., `<AppDomain>https://myappdomain.com</AppDomain>`).
+> 2. If there is an explicit port for the domain, this must be included too (e.g.,`<AppDomain>https://myappdomain.com:9999</AppDomain>`).
```

Comment:
Maybe just "If there is an explicit port, include it" or similar. (Same with the next line below, #3.) Imo the "too" looks informal to me